taskt/Core/Automation/Commands/Data Commands/SplitTextCommand.cs

```diff
@@ -1,110 +1,116 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Xml.Serialization;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to split a string")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to split a single text or variable into multiple items")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command uses the String.Split method to achieve automation.")]
+    [Group("Data Commands")]
+    [Description("This command splits a string by a delimiter and saves result in a list.")]
     public class SplitTextCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select variable or text to split")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_userVariableName { get; set; }
+        [PropertyDescription("Text Data")]
+        [InputSpecification("Provide a variable or text value.")]
+        [SampleUsage("Sample text, to be splitted by comma delimiter || {vTextData}")]
+        [Remarks("Input data other than string (text) type will cause error.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_InputText { get; set; }
+
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Input Delimiter (ex. [crLF] for new line, [chars] for each char, ',')")]
-        [Attributes.PropertyAttributes.InputSpecification("Declare the character that will be used to seperate. [crLF] can be used for line breaks and [chars] can be used to split each digit/letter")]
-        [Attributes.PropertyAttributes.SampleUsage("[crLF], [chars], ',' (comma - with no single quote wrapper)")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_splitCharacter { get; set; }
+        [PropertyDescription("Text Delimiter")]
+        [InputSpecification("Specify the character that will be used to split the text.")]
+        [SampleUsage("[crLF] || [chars] || ,")]
```

Comment:
Are these the only 3 options? If so, maybe a dropdown. If not, please add a {vDelimiter} option

Reply:
Added {vDelimiter}